File: unittest/scripts/devapi/scripts/working_with_results.js
// Assumptions: test schema assigned to db, my_collection collection exists

// Gets the collection
var myColl = db.getCollection('my_collection');

// Insert a document
var res = myColl.add({ name: 'Jack', age: 15, height: 1.76 }).execute();

// Print the documentId that was assigned to the document
print('Document Id:', res.getLastDocumentId());

File: unittest/scripts/py_dev_api_examples/results/Working_with_Results.py
# Gets the collection
myColl = db.get_collection('my_collection')

# Insert a document
res = myColl.add({'_id': '00001', 'name': 'Jack', 'age': 15, 'height': 1.76 }).execute()

#@ Print the documentId that was assigned to the document {VER(>=8.0.5)}
print 'Document Id: %s\n' % res.get_generated_ids()[0]
